src/shared/constants/chart-props.js

```diff
@@ -81,3 +81,19 @@ export const barChartDefaultProps = {
   groupByKey: '',
   valueKey: '',
 }
+
+export const typographyPropTypes = {
+  typographyProps: PropTypes.shape({
+    fontFamily: PropTypes.string.isRequired,
+    fontSize: PropTypes.number.isRequired,
+    textColor: PropTypes.string.isRequired
+  })
+}
+
+export const typographyDefaultProps = {
+  typographyProps: {
+    fontFamily: 'Open Sans',
```

Comment:
make sure to give sufficient fallback fonts as well (bottomline should have a `sans-serif` at the end):

```js
  fontFamily: '"OpenSans", Helvetica, sans-serif',
```